src/Site.elm

```diff
@@ -33,7 +34,7 @@ data =
 head : Data -> List Head.Tag
 head static =
     [ Head.sitemapLink "/sitemap.xml"
-    , Head.appleTouchIcon (Just 180) (pathFromString "/favicons/apple-touch-icon.png")
+    , Head.appleTouchIcon (Just 180) (fullPathFromString "/favicons/apple-touch-icon.png")
```

Comment:
Did this used to produce a relative url?
Is full domain equivalent to and external url?

Reply:
ah ok, I started adding an `AbsoluteInternal` type to account for this situation then I realized I was in the library code not our own so I shouldn't meddle with it. A bit strange that elm-pages makes that decision for you up front about how to format internal URLs